Source/CCZ4/GammaCalculator.hpp

```diff
@@ -38,12 +38,13 @@ class GammaCalculator
   public:
     GammaCalculator(double a_dx) : m_deriv(a_dx) {}
 
-    template <class data_t> void compute(Cell<data_t> current_cell) const
+    void compute(Cell current_cell) const
     {
         // copy data from chombo gridpoint into local variables, and calc 1st
         // derivs
-        const auto vars = current_cell.template load_vars<Vars>();
-        const auto d1   = m_deriv.template diff1<Vars>(current_cell);
+        Vars<amrex::Real> vars;
+        load_vars(current_cell, vars);
```

Comment:
This code hasn't been ported yet so maybe we should add a comment that we've
just removed the templating
```suggestion
        // TODO: Port this code
        // This code hasn't been ported to GRTeclyn.
        // We've just removed templating over data_t.
        Vars<amrex::Real> vars = load_vars(current_cell);
```